generators/generator-base.js

```diff
@@ -2457,13 +2457,14 @@ templates: ${JSON.stringify(existingTemplates, null, 2)}`;
 
     dest.testFrameworks = config.testFrameworks || [];
 
-    dest.gatlingTests = dest.testFrameworks.includes(GATLING);
-    dest.cucumberTests = dest.testFrameworks.includes(CUCUMBER);
-    dest.protractorTests = dest.testFrameworks.includes(PROTRACTOR);
-    dest.cypressTests = dest.testFrameworks.includes(CYPRESS);
+    dest.gatlingTests = config.testFrameworks.includes(GATLING);
+    dest.cucumberTests = config.testFrameworks.includes(CUCUMBER);
+    dest.protractorTests = config.testFrameworks.includes(PROTRACTOR);
+    dest.cypressTests = config.testFrameworks.includes(CYPRESS);
```

Comment:
```suggestion
    const testFrameworks = config.testFrameworks || [];
    dest.gatlingTests = testFrameworks.includes(GATLING);
    dest.cucumberTests = testFrameworks.includes(CUCUMBER);
    dest.protractorTests = testFrameworks.includes(PROTRACTOR);
    dest.cypressTests = testFrameworks.includes(CYPRESS);
```
dest was used due to a safeguard for undefined.